lib/anaconda/form_builder_helpers.rb

```diff
@@ -81,7 +87,6 @@ def anaconda( anaconda_field_name, form_options = {} )
       output += <<-END
 <div id="#{upload_details_container_id}" class="anaconda_upload_details_container"></div>
 <script>
-  document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Did you intend to remove this? This is a typical callback that's used to ensure that the DOM is ready to execute any scripts that may modify elements expected to be present. Otherwise, those scripts can fail if they DOM isn't "ready."

Reply:
I did not. Sorry. Moving too quickly to try and get it in before the EOD and missed it.